### DISCUSSION TITLE
Allow labels in two languages

### DISCUSSION THREAD

**Discussion by spacekookie** [osmandapp/OsmAnd#17684] [Ideas]
Currently you can either have place labels in the native language, or have them translated to a different language.

It would be great if OsmAnd could show both the native name, as well as a translation. Especially for non-Latin scripts. Other maps applications already do this, including qwant maps, if you wanna have a look at design ideas and what data to rely on from a POI.

It would navigating in places that that use different scripts much easier, because you can still try to match the shape of a word, without having to understand what exactly it means.

**Comment by vshcherb** [comment]
Yes, it's a good idea, we are thinking to implement it.
